browser/src/TeamPage/TeamPage.js

```diff
@@ -0,0 +1,277 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { PageHeading } from '@gnomad/ui'
+
+// JSON containing information for all members with bios and headshots
+import teamMembers from './TeamMembers.json'
+
+// Members of the "Staff" section
+import websiteStaff from '../../about/contributors/website.md'
+import productionStaff from '../../about/contributors/production.md'
+import operationsStaff from '../../about/contributors/operations.md'
+
+// Members of the "Contributors" section
+import dataGenerationContributors from '../../about/contributors/data-generation.md'
+import structuralVariationContributors from '../../about/contributors/structural-variation.md'
+import mitochondrialVariationContributors from '../../about/contributors/mitochondrial-variation.md'
+import broadGenomicsPlatformContributors from '../../about/contributors/broad-genomics-platform.md'
+import ethicsContributors from '../../about/contributors/ethics.md'
+
+// TODO: analysis import - when members are finalized
+// TODO: operations import - when members are finalized
+
+// Members of the 'Alumni' Section
+import alumni from '../../about/contributors/alumni.md'
+
+import DocumentTitle from '../DocumentTitle'
+import InfoPage from '../InfoPage'
+
+const Team = styled.div`
+  padding-bottom: 1rem;
+`
+const TeamSection = styled.div`
+  padding-bottom: 1rem;
+  margin-bottom: 1rem;
+`
+const TeamSectionList = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 80%;
+  padding: 1rem;
+  margin: 1rem;
+`
+
+/**
+ * Used for a responsive 3 column layout for contributors
+ */
+const ColumnList = styled.div`
+  content: '';
+  display: table;
+  width: 100%;
+  clear: both;
+`
+
+const ResponsiveColumn = styled.div`
+  float: left;
+  width: 33.33%;
+
+  @media screen and (max-width: 600px) {
+    width: 100%;
+  }
+`
+
+// Component and Styling for a Card
+// Is there better practice between taking in a single (prop) with fields, or with
+//   taking in the destructured fields of a given prop/object?
+const TeamCard = ({ title, description, headshotSource }) => {
+  return (
+    <TeamHeadshotAndDescription>
+      <Row>
+        <ImageColumn>
+          <Headshot alt="headshot" src={headshotSource} />
```

Comment:
Here its used in a styled `<img>` tag, with the source attribute. This was the best way I could figure to be able to map over the data and keep this component clean while having the data be easily adjustable. Is there a better way to deal with the headshot images?